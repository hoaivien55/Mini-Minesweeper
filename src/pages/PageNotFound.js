import { Link } from "react-router-dom";
import { Container, Row } from "../components/styles";

const PageNotFound = () => {
    return <Container>
        <Row style={{fontSize: '70px'}}>404</Row>
        <Row>Page not found.</Row>
        <Link to="/">Home page</Link>
    </Container>
}

export default PageNotFound;