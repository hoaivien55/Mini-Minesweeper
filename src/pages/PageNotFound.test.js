import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

import PageNotFound from "./PageNotFound";

it("PageNotFound: Virtual DOM snapshot", () => {
	const tree = renderer
		.create(
			<BrowserRouter>
				<PageNotFound />
			</BrowserRouter>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
