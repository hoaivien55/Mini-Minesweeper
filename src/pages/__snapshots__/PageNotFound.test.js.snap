// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound: Virtual DOM snapshot 1`] = `
.c0 {
  position: absolute;
  left: 50%;
  top: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  margin: 0 auto;
  width: auto;
  min-width: 300px;
  min-height: 200px;
  padding: 20px;
  background-color: white;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0px;
}

<div
  className="c0"
>
  <div
    className="c1"
    style={
      Object {
        "fontSize": "70px",
      }
    }
  >
    404
  </div>
  <div
    className="c1"
  >
    Page not found.
  </div>
  <a
    href="/"
    onClick={[Function]}
  >
    Home page
  </a>
</div>
`;
